test/Dialect/FIRRTL/parse-basic.fir

```diff
@@ -44,6 +44,14 @@ circuit MyModule :     ; CHECK: firrtl.circuit "MyModule" {
     parameter DEFAULT = 0
     parameter WIDTH = 32
     parameter DEPTH = 32.42
+    
+  ; Check that integers are extended to 32 bits if they are smaller.
+  ; CHECK-LABEL: firrtl.extmodule private @IntegerParamsModul
+  extmodule IntegerParamsModule :
+    ; CHECK-SAME: a: ui32 = 1
+    parameter a = 1
+    ; CHECK-SAME: b: ui64 = 2251799813685248
```

Comment:
Does it matter what width we create this at?  Seems we round up to next power-of-two, while FIRRTL maybe uses minimum width needed but not more (here, 52 bits).  Seems easy to tweak if we care while visiting this.